charts/azad-kube-proxy/templates/deployment.yaml

```diff
@@ -70,6 +70,26 @@ spec:
       {{- end }}
       {{- with .Values.affinity }}
       affinity:
+        podAntiAffinity:
```

Comment:
Can't we define this in the values.yaml? If not we should remove the with .Values.affinity above.

Reply:
It's possible. After a discussion with @phillebaba we decided that it's better to be opinionated in this case.
We don't need all these config values.
This anti-affinity rule will work on any cloud assuming that they follow topologyKey: topology.kubernetes.io/zone (all the big 3 ones does).

Reply:
Missed to remove "{{- with .Values.affinity }}".
Yes we can be we thought opinion was better